ansible_collections/arista/avd/roles/eos_designs/templates/base/router-multicast.j2

```diff
@@ -0,0 +1,6 @@
+{% if switch.evpn_multicast is arista.avd.defined(true) %}
+router_multicast:
+  ipv4:
+    routing: true
```

Comment:
I think we forgot to do the first three lines of this based on `underlay_multicast`.
Lets refactor this one to:
```jinja
{% if switch.underlay_multicast is arista.avd.defined(true) %}
router_multicast:
  ipv4:
    routing: true
{%     if switch.evpn_multicast is arista.avd.defined(true) %}
    software_forwarding: sfe
{%     endif %}
{% endif %}
```

Reply:
good catch, fixed!